### PR TITLE
fix: Do not duplicate event entries when searching.

### LINKED ISSUE #944: Duplicated disnake.disnake in Event Reference for commands framework in docs
### Summary

Search page results and link fragment identifiers (e.g. disnake.disnake.ext.commands.on_slash_command_error)

### Reproduction Steps

https://docs.disnake.dev/en/latest/search.html?q=disnake.disnake
Issue persists for stable docs version as well

image

### Minimal Reproducible Code

_No response_

### Expected Results

Normal fragment identifiers without duplicated `disnake.disnake`, e.g. `disnake.ext.commands.on_slash_command_error`

### Actual Results

Duplicated fragment identifiers, e.g. `disnake.disnake.ext.commands.on_slash_command_error`

### Intents

Unrelated

### System Information

```markdown
Unrelated
```

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by ItsAleph**
## Summary

Closes #944 

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `task lint`
    - [ ] I have type-checked the code by running `task pyright`
- [x] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by shiftinv** [comment]
This doesn't seem to resolve the original issue: 
image

**Comment by ItsAleph** [comment]
Looks like Sphinx did some implications or assumptions for base object path due to all other objects being defined with `disnake.ext.commands.` prefix. I think this is resolved now.

File: docs/ext/commands/api.rst [hunk 1]
```diff
@@ -1095,12 +1095,12 @@ Exception Hierarchy
...
+.. autoclass:: MessageContentPrefixWarning
 
 Warning Hierarchy
 ~~~~~~~~~~~~~~~~~~~
 
 .. exception_hierarchy::
 
-    - :class:`DiscordWarning`
+    - :class:`~.DiscordWarning`
         - :class:`~.commands.MessageContentPrefixWarning`
```
Comment (onerandomusername) [inline docs/ext/commands/api.rst:1106]: I believe this should be 
```suggestion
        - :class:`~.MessageContentPrefixWarning`
```

to fit in with the rest of the changes

Comment (ItsAleph) [reply docs/ext/commands/api.rst]: Didn't have any impact except for the aesthetics, perhaps. Thanks!

**Comment by shiftinv** [comment]
This change results in the `Bot.on_message_command_error` description now linking to `commands.on_slash_command_error` instead of `Bot.on_slash_command_error` like before; same with `Bot.on_user_command_error`.

other than that, lgtm

**Comment by ItsAleph** [comment]
> This change results in the `Bot.on_message_command_error` description now linking to `commands.on_slash_command_error` instead of `Bot.on_slash_command_error` like before; same with `Bot.on_user_command_error`.

Good catch, thanks! This is fixed now.

**Comment by ItsAleph** [comment]
Oh wait it's not

**Comment by onerandomusername** [comment]
> Oh wait it's not

It is fixed for me.

**Comment by ItsAleph** [comment]
> It is fixed for me.

Oh, well, that must be caching i guess
~~https://shouldiblamecaching.com/~~

**Review by shiftinv — APPROVED** [review]

**Pull request merged**